src/main/java/janggi/domain/piece/behavior/rotatemove/OrthogonalRotateMoveBehavior.java

```diff
@@ -29,13 +33,19 @@ public final Set<Position> generateAvailableMovePositions(Board board, Side side
 
     protected abstract void searchAvailableMoves(Set<Position> result, Board board, Position currentPosition,
                                                  List<Vectors> vectorsList,
-                                                 Side side);
+                                                 Team team);
 
-    protected abstract void searchAvailableMove(Set<Position> result, Board board, Position currentPosition, Side side,
+    protected abstract void searchAvailableMove(Set<Position> result, Board board, Position currentPosition, Team team,
                                                 Vectors vectors);
 
     protected boolean canNotMove(Vectors vectors, Position currentPosition) {
-        return vectors.vectors()
+        int size = vectors.vectors().size();
```

Comment:
아래 방식이 더 깔끔하지 않을까요~?
```suggestion
        int size = vectors.size();
```

Reply:
추가하였습니다!